contracts/utils/MathMock.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { Math } from './Math.sol';
+
+contract MathMock {
+    using Math for uint256;
+
+    function average(uint256 _a, uint256 _b) external pure returns (uint256) {
+        return _a.average(_b);
+    }
+
+    function sqrt(uint256 _x) external pure returns (uint256) {
+        return _x.sqrt();
```

Comment:
Don't use underscore in variable names.  That convention is useful for distinguishing between storage variables and function arguments, but the SolidState library does not use storage variables anywhere.

Reply:
I thought it was to distinguish between function arguments and any other variable, such as those initialized in the function body. 

Noted. Thank you.